mix.exs

```diff
@@ -57,12 +58,15 @@ defmodule Appsignal.Mixfile do
   defp test_paths(:test_phoenix), do: ["test/appsignal", "test/phoenix"]
   defp test_paths(_), do: ["test/appsignal"]
 
+  defp elixirc_paths(:test), do: ["lib", "test/support"]
```

Comment:
Looks like this would fix the failing tests

```elixir
defp elixirc_paths(env) when env in [:test, :test_phoenix, :test_no_nif], do: ["lib", "test/support"]
```